server.js

```diff
@@ -26,12 +26,14 @@ app.get('/location', (request, response) => {
 })
 
 app.get('/weather', (request, response) => {
-  // superagent.get(`https://api.darksky.net/forecast/${process.env.DARKSKY_API_KEY}/${request.query.data.latitude},${request.query.data.longitude}`).then(result => {
-  //   console.log(result.body);
-  // })
-  response.send(search_weather());
+  superagent.get(`https://api.darksky.net/forecast/${process.env.DARKSKY_API_KEY}/${request.query.data.latitude},${request.query.data.longitude}`).then(result => {

```

Comment:
You may be getting the data back from darksky.net, but you are not using it. I can see you started writing a helper function for this below, which is what you want to call here instead. Of course you probably want some version of this code line inside your helper function. I suspect you just haven't gotten to this yet.